data.json

```
@@ -0,0 +1,6 @@
+{
+    "한식": ["국밥", "갈비탕", "냉면", "치킨", "해장국", "순두부찌개", "김치찌개", "부대찌개", "떡볶이", "비빔밥", "된장", "감자탕", "칼국수", "설렁탕", "보쌈", "김원준"],
```

```suggestion
    "한식": ["국밥", "갈비탕", "냉면", "치킨", "해장국", "순두부찌개", "김치찌개", "부대찌개", "떡볶이", "비빔밥", "된장", "감자탕", "칼국수", "설렁탕", "보쌈"],
```
음식이 아닌 내용은 제외합시다 ㅎㅅㅎ